leica_streaming_app/LeicaPoint[] points
